demoscene/utils/groklinks.py

```diff
@@ -1710,6 +1710,14 @@ def __str__(self):
         return u"https://diskmag.conspiracy.hu/magazine/%s/#/edition=%s" % (magazine, edition)
 
 
+class EventsRetrosceneRelease(AbstractBaseUrl):
+    site =  Site("Events", long_name="Retroscene events", icon_path='images/icons/external_sites/events_retroscene.png')
```

Comment:
You haven't specified a `url` or `allowed_hostnames` argument here, so this ends up matching every URL, not just ones on events.retroscene.org.

Reply:
Added `url` argument